python/src/scipp/plot/figure.py

```diff
@@ -46,12 +44,25 @@ def __init__(self,
         self.axlocator = {}
 
     def savefig(self, filename=None):
+        """
+        Save plot to file.
+        Possible file extensions are `.jpg`, `.png` and `.pdf`.
```

Comment:
It may be worth providing information on what the default save directory is here as well.